packages/notebook/src/browser/variables/variable-view.tsx

```
@@ -0,0 +1,312 @@
+import { Input, List, Tag, Tooltip, message } from 'antd';
+import React, { forwardRef } from 'react';
+import './index.less';
+
+import { localize } from '@opensumi/ide-core-common';
+
+import { Languages } from './inspector-script';
+
+import type { LanguageModel } from './inspector-script';
+import type { KernelMessage } from '@difizen/libro-kernel';
+
+const LibroVariableComponent = forwardRef<HTMLDivElement, {}>((props, ref) => {
```

Biome (1.9.4)

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)